tests/integ/test_udf.py

```diff
@@ -1955,3 +1955,14 @@ def echo(num: int) -> int:
         TestData.all_nulls(session).to_df(["a"]).select(echo("a")),
         [Row(None), Row(None), Row(None), Row(None)],
     )
+
+
+def test_secure_udf(session):
+    @udf(secure=True)
```

Comment:
Is there a way to check whether the created udf is a secured udf?

Reply:
I didn't see this in `describe`, but I probably can check if `secure` exists in `get_ddl`. Let me give it a try. Good callout.